src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -315,13 +315,16 @@ def verify_io_dtx_committed_metrics(self, baseline_data, metrics_data, test_metr
         # test ops = (create dir + punch obj + punch dir) * repetitions
         test_ops *= repetitions
         num_operations += test_ops
+        # 2 DTX entries on 2 replicas
+        test_servers = self.params.get("test_servers", "/run/*")
+        num_operations *= test_servers
```

Comment:
The better way to get the number of server hosts is to use the class that manages starting the servers, but I suspect you want engine count here:
```suggestion
        num_operations *= self.server_managers[0].engines
```
Instead, if you do really need the number of hosts:
```
        num_operations *= len(self.server_managers[0].hosts)
```